README.md

```diff
@@ -932,7 +932,7 @@ The following table lists the parameters for the `redis` component and their def
 
 | Parameter | Description | Default |
 | -------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
-| `redis.enabled` | Enable the component | `true` |
+| `redis.enabled` | Enable the component | `nil` |
```

Comment:
Good catch, but it's actually `false`: https://github.com/mendersoftware/mender-helm/blob/master/mender/values.yaml#L1414

```suggestion
| `redis.enabled` | Enable the component | `false` |
```